Dockerfile

```diff
@@ -40,7 +40,13 @@ RUN apt-get update \
         zlib1g-dev \
     && rm -rf /var/lib/apt/lists/*
 
-RUN python3 -m pip install -U pip
+#RUN apk add --update --no-cache python3 && ln -sf python3 /usr/bin/python
+#RUN python3 -m ensurepip
+RUN rm -d -rf /usr/share/doc/openjdk-17-jre-headless
+RUN apt update -y && apt upgrade -y && apt dist-upgrade -y
+RUN apt install -y python3.9-venv
+RUN mkdir /venv
+RUN python3 -m venv /ssm/venv
```

Comment:
If you add python-is-python3 in the apt install, you can just use `python` instead.

Thoughts?